src/libs/protocols/protos/agent.proto

```diff
@@ -295,6 +295,9 @@ message CreateSandboxRequest {
 	string guest_hook_path = 6;
 	// This field is the list of kernel modules to be loaded in the guest kernel.
 	repeated KernelModule kernel_modules = 7;
+	// This fieldis the number of seconds to wait for a hook execution before aborting it.
+	// It must be a non-negative integer. Zero means no timeout. The default value is 10 seconds.
+	int32 guest_hook_timeout = 8;
```

Comment:
nit: If the guest hook is a `non-negative` integer, why we are using `int32` here?

Reply:
https://github.com/kata-containers/kata-containers/blob/main/src/libs/oci/src/lib.rs#L188
according to the Hook struct defined in Rust, it also used `i32`, so I think it should be consistent.